DualityEditorPlugins/EditorBase/PropertyEditors/Resources/FontContentPropertyEditor.cs

```diff
@@ -27,7 +27,7 @@ protected override void OnGetValue()
 			Font font = this.GetValue().FirstOrDefault() as Font;
 
 			bool lastCanRenderFont = this.canRenderFont;
-			this.canRenderFont = (font != null && font.EmbeddedTrueTypeFont != null);
+			this.canRenderFont = (font != null && font.GetGlyphBitmap(' ') != null);

```

Comment:
> Can you suggest another way to fix this?

In the end, it will be fixed by me implementing issue #285 and updating Font accordingly, but I don't know when that will be yet. So, in the meantime, one way to deal with this would be to further customize the font "content" property editor so it makes a distinction between "TTF rendering" and "Duality data" properties, and only makes the TTF rendering ones readonly. 

This will of course be a bit hacky, but since this will be replaced at some point anyway, I think it should be fine. I can't really point you towards a specific solution, but I think it can be implemented in the existing [FontContentPropertyEditor](https://github.com/AdamsLair/duality/blob/master/DualityEditorPlugins/EditorBase/PropertyEditors/Resources/FontContentPropertyEditor.cs) provided you configure it using the special case in [FontPropertyEditor](https://github.com/AdamsLair/duality/blob/master/DualityEditorPlugins/EditorBase/PropertyEditors/Resources/FontPropertyEditor.cs).